src/pooler.erl

```diff
@@ -202,13 +203,21 @@ take_member(PoolName) when is_atom(PoolName) orelse is_pid(PoolName) ->
 take_member(PoolName, Timeout) when is_atom(PoolName) orelse is_pid(PoolName) ->
     gen_server:call(PoolName, {take_member, time_as_millis(Timeout)}, infinity).
 
-
 %% @doc Take a member from a randomly selected member of the group
 %% `GroupName'. Returns `MemberPid' or `error_no_members'.  If no
 %% members are available in the randomly chosen pool, all other pools
 %% in the group are tried in order.
 -spec take_group_member(atom()) -> pid() | error_no_members | {error_no_group, atom()}.
-take_group_member(GroupName) ->
+take_group_member(GroupName) -> take_group_member(GroupName, 0).
+
+%% @doc Take a member from a randomly selected member of the group
+%% `GroupName'. Returns `MemberPid' or `error_no_members'.  If no
+%% members are available in the randomly chosen pool, all other pools
+%% in the group are tried in order. If no member
+%% is available within the specified timeout, error_no_members is returned.
+%% `Timeout' can be either milliseconds as integer or `{duration, time_unit}'
+-spec take_group_member(atom(), non_neg_integer()) -> pid() | error_no_members | {error_no_group, atom()}.
```

Comment:
The spec here for `Timeout` doesn't include `{duration(), time_unit()}` -- if we're going to support that, the spec should allow it as well.